build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.3.3'
-        classpath 'com.novoda:bintray-release:0.5.0'
+        classpath 'com.novoda:bintray-release:0.8.0'
```

Comment:
https://stackoverflow.com/questions/47470631/no-such-property-for-runtime-for-class-org-gradle-api-attributes-usage/49546085#49546085